### ISSUE TITLE
Assign Zenodo DOI

### ISSUE THREAD

**Issue by gwaybio** [greenelab/GEA_Community_Detection#26]
Once public, we will need to generate a zenodo DOI for the repository by following instructions

https://guides.github.com/activities/citable-code/

Also, need to add DOI badge to README.

**Comment by cgreene** [comment]
GitHub linked so all new releases will have a DOI assigned. The DOI associated with the PSB paper is:
http://doi.org/10.5281/zenodo.830568

DOI not yet resolving - here's the direct zenodo link:
https://zenodo.org/record/830568

**Comment by gwaygenomics** [comment]
alright! Thanks @cgreene - looks like the DOI is resolving already. I will file a PR to get the svg button into the README. @lharr - can you add the citation in the text?

**Comment by lharr** [comment]
Yes, I'll add the citation! I'm assuming it's the http://doi.org/10.5281/zenodo.830568 version to put in?

**Comment by cgreene** [comment]
I think this is the citation (from the side of Zenodo) - may need light formatting to match PSB style

Lia X. Harrington, Gregory P. Way, Jennifer A. Doherty, & Casey S. Greene. (2017, July 17). Gene Enrichment Analysis via Community Detection. Zenodo. http://doi.org/10.5281/zenodo.830568
